.github/workflows/vast.yaml

```diff
@@ -15,6 +15,9 @@ on:
     types:
       - opened
       - synchronize
+  merge_group:
+    types:
+      - checks_requested
```

Comment:
Below this we have the concurrency.cancel_in_progress section. We should consider also enabling that for merge queues.